OLS-2400: Document support for OLS API (#110626)
openshift/openshift-docs: modules/ols-proc-expose-the-ols-service-by-using-a-route.adoc

// Module included in the following assemblies:
// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc

:_mod-docs-content-type: PROCEDURE
[id="ols-proc-expose-the-ols-service-by-using-a-route_{context}"]
= Expose the {ols-long} service by using a route

> By default, the {ols-long} service is a `ClusterIP` type and is only reachable from inside the cluster. To access the {ols-long} REST API from external tools or scripts, you must create an {ocp-short-name} route.

Reviewer: ```suggestion

The {ols-long} service is an internal-only `ClusterIP` type by default. To enable external access to the {ols-long} REST API, you must create an {ocp-short-name} route.
```
